### ISSUE TITLE
Pull requests API 

### ISSUE THREAD

**Issue by mayra-cabrera** [IcaliaLabs/watcher#13]
**This should be done after #12 has been merged into master**
____

It should have four endpoints

### 1. Total, open & closed pull requests for an organisation 

One that returns total, open and  closed pull requests for an specific organization.
i.e.:

```ruby
# http://localhost:3000/api/v1/organizations/icalialabs/pull_requests_info
{"total_pull_requests":300,"open_pull_requests":100,"closed_pull_requests":200}
```

Please notice that

- `icalialabs` is a parameter
- This method should be inside organizations controller 

### 2. All pull requests

Another one that returns all pull requests from a repository in an specific organisation 
i.e.:
```ruby
# Returns all pull requests for furatto
http://localhost:3000/api/v1/repos/icalia_labs/furatto/pull_requests
```

### 3. Open pull requests  for a repository 

Another one that returns open pull requests from a repository in an specific organisation

```ruby
# Returns open pull requests for furatto
http://localhost:3000/api/v1/repos/icalia_labs/furatto/open_pull_requests
```

### 4. Closed pull requests for a repository

i.e.:
```ruby
# Returns closed pull_requests for furatto
http://localhost:3000/api/v1/repos/icalia_labs/furatto/closed_pull_requests
```

For 2, 4 & 4 notice that:

-  `icalia_labs` and `furatto` are parameters that are sent to repos controller 
- Pull requests objects must contain the following information: 

```ruby
{
"id": 1,
"github_url": "https://github.com/icalialabs/furatto/issues/1234",
"number": "1234",
"title": "Found a bug",
"body": "I'm having a problem with this.",
"author": "kurenn",
"labels": ["bug"],
"assignee": "mayra-cabrera" #If there's one assignee
"created_at": "2011-04-22T13:33:48Z",
}
```
